petastorm/spark/spark_dataset_converter.py

```diff
@@ -200,6 +200,8 @@ def make_tf_dataset(
             prefetch=None,
             num_epochs=None,
             workers_count=None,
+            shuffle_buffer_size=None,
```

Comment:
Is `shuffle_buffer_size` effectively different from `shuffling_queue_capacity` in `make_torch_dataloader`? If not, we should probably call them the same.

Reply:
Sounds good.  Do you want to go with `shuffling_queue_capacity` since it's already being used by `make_torch_dataloader`?